internal/controller/capacity.go

```diff
@@ -127,6 +136,11 @@ func (*InstasliceReconciler) getStartIndexFromPreparedState(instaslice *inferenc
 			}
 		}
 	}
+	return gpuAllocatedIndex[:]
```

Comment:
why do we need to convert the fixed array into a slice?

Reply:
I left a comment there, returning a slice to minimize expense instead of returning copy of the array and also for flexibility fo size later on as we are now generalizing size 8 for all GPUs

Reply:
I like to have a more readable code than having to optimize for an array of length eight, please return back the array and not the slice